eng/Build.props

```
@@ -126,9 +126,10 @@
           the entire pattern will silently fail to evaluate correctly.
         -->
         <DotNetProjects Include="
-                          $(RepoRoot)src\Framework\ref\Microsoft.AspNetCore.App.Ref.csproj;
-                          $(RepoRoot)src\Framework\src\Microsoft.AspNetCore.App.Runtime.csproj;
-                          $(RepoRoot)src\Framework\test\Microsoft.AspNetCore.App.UnitTests.csproj;
+                          $(RepoRoot)src\Framework\Public\ref\Microsoft.AspNetCore.App.Ref.csproj;
+                          $(RepoRoot)src\Framework\Public\src\Microsoft.AspNetCore.App.Runtime.csproj;
+                          $(RepoRoot)src\Framework\Internal\ref\Microsoft.AspNetCore.App.Ref.Internal.csproj;
+                          $(RepoRoot)src\Framework\Public\test\Microsoft.AspNetCore.App.UnitTests.csproj;
```

This should be fine but
1. Why bother moving the single test/ folder?
2. Would it be better to make the layout a bit more consistent with other src/ folders e.g.
```suggestion
                          $(RepoRoot)src\Framework\App.Ref\src\Microsoft.AspNetCore.App.Ref.csproj;
                          $(RepoRoot)src\Framework\App.Runtime\src\Microsoft.AspNetCore.App.Runtime.csproj;
                          $(RepoRoot)src\Framework\App.Ref.Internal\src\Microsoft.AspNetCore.App.Ref.Internal.csproj;
                          $(RepoRoot)src\Framework\test\Microsoft.AspNetCore.App.UnitTests.csproj;
```